cli/js/error_stack.ts

```diff
@@ -256,9 +258,13 @@ function prepareStackTrace(
           return callSite;
         }
       )
-      .map((callSite): string => `    at ${callSiteToString(callSite)}`)
-      .join("\n")
-  );
+      .map((callSite): string => {
+        // @ts-ignore
+        error["__callSites"].push(callSite); // Save callsites to self
```

Comment:
Maybe just assign the whole array above/below?

Reply:
I can definitely do that, but actually I was thinking about not saving `callSite` but an object populated with data we need (since currently I am actually calling a bunch of function from Rust)

Reply:
I agree with @kevinkassimo on this one.

Reply:
I would also consider implementing the entire prepareStackTrace function in C++.

Not 100% sure, but it might be nice to keep the part where the call sites get stored and where they get processed close together.

That is something that I can do if you don't want to btw.

Reply:
@piscisaureus Yeah I think that is reasonable. I don't think I will have enough time to work on it this week (finals are coming), so yeah for sure